readme.txt

```diff
@@ -36,8 +36,8 @@ WooCommerce Admin also allows store owners to customize a new dashboard screen w
 
 = Minimum Requirements =
 
-* WordPress 5.3
-* WooCommerce 4.5 or greater
+* WordPress 5.4
+* WooCommerce 4.8 or greater
 * PHP version 7.0 or greater. PHP 7.2 or greater is recommended
```

Comment:
its interesting to me that our readme says php 7.0 but our plugin file says php 5.6.20.

I think woo core also says php 5.6.20 on their plugin and I *think* thats still the min supported version by Wordpress. should we change this to reflect that?

Reply:
We bumped the PHP version to 7.0 recently [here](https://github.com/woocommerce/woocommerce-admin/pull/6046), but it looks like I missed the version number in the plugin file.

Reply:
Thanks @adrianduffell I missed that wc had updated that!

Reply:
Good spotting, thank you! Fixed in https://github.com/woocommerce/woocommerce-admin/pull/6342/commits/73a1bf7fc7627327f57c4498e6c6c29b96790d48